services/bundlejs/bundlejs-package.service.js

```diff
@@ -110,7 +105,7 @@ export default class BundlejsPackage extends BaseJsonService {
 
   async handle({ scope, packageName }, { exports }) {
     const json = await this.fetch({ scope, packageName, exports })
-    const size = json.size.compressedSize
-    return this.constructor.render({ size })
+    const size = json.size.rawCompressedSize
+    return renderSizeBadge(size, 'metric', 'minified size (gzip)')
```

Comment:
Previously we were presenting `compressedSize` as reported by the upstream
Switching this to use `rawCompressedSize` allows us to apply our own rounding convention while matching the upstream units